Language.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Language.h";
+
+std::map<std::string, std::string> langfile;
+
+void loadLanguageFile(std::string filename) {

```

Comment:
Neat parsing. However, users will need to be careful to avoid leading spaces, spaces around the `=`, etc. In other words, 'learn' how to use your file.
Try using json. There are header-only libraries for interpreting json in c++, like https://github.com/nlohmann/json